torch/fx/operator_schemas.py

```diff
@@ -124,26 +134,34 @@ def type_matches(signature_type : Any, argument_type : Any):
         # int can be promoted to List[int]
         return True
 
-    def is_homogeneous_int_tuple(t):
-        if not getattr(t, '__origin__', None) in {tuple, Tuple}:
-            return False
+    if getattr(signature_type, '__origin__', None) in {list, List}:
+        sig_el_type = signature_type.__args__[0]
+        if not inspect.isclass(sig_el_type):
+            raise RuntimeError("Does not currently support nested parametric types")
```

Comment:
Can you add some debug information here, e.g. the value of `signature_type` and direct the user to file a bug to PyTorch?